matplotcheck/base.py

```diff
@@ -909,19 +915,21 @@ def assert_xydata(
             xy_expected.sort_values(by=xcol),
         )
 
-        if tolerence > 0:
+        if tol_rel > 0 or tol_abs > 0:
```

Comment:
once we figure this out it does seem like we should do a quick check to ensure the user provides a reasonable tolerance value. does assert_allclose check that the tolerance value is a number?